src/scripts/assume-role-with-web-identity.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+PARAM_ROLE_SESSION_NAME=$(eval echo "${PARAM_ROLE_SESSION_NAME}")
+
+# shellcheck disable=SC2086,SC2034
+read -r AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY AWS_SESSION_TOKEN <<<"$(aws sts assume-role-with-web-identity \
+    --role-arn ${PARAM_AWS_CLI_ROLE_ARN} \
+    --role-session-name ${PARAM_ROLE_SESSION_NAME} \
+    --web-identity-token ${CIRCLE_OIDC_TOKEN} \
+    --duration-seconds ${PARAM_SESSION_DURATION} \
+    --query 'Credentials.[AccessKeyId,SecretAccessKey,SessionToken]' \
+    --output text)"
+
+{
```

Comment:
In reference to my top comment about not ignoring `SC2129`

Reply:
Thanks, I removed `SC2129` from the exclude list